### PR TITLE
Prepare for v6 release

### LINKED ISSUE #54: Compatibility with Autofac v6
v6 Compatibility and version dependency checks ahead of release.

### PR DESCRIPTION
**Pull request by alexmg**
- Bump package version to `4.0.0`
- Bump `net462` target to `net472`
- Build and project standardization
- Update ServiceFabric package to latest

### CODE REVIEW

File: src/Autofac.Integration.ServiceFabric/Autofac.Integration.ServiceFabric.csproj [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
   <PropertyGroup>
     <Description>Autofac integration for Azure Service Fabric. Provides service factory implementations for Actors, Stateful Services and Stateless Services.</Description>
-    <VersionPrefix>3.0.0</VersionPrefix>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;net461</TargetFrameworks>
+    <!-- VersionPrefix patched by AppVeyor -->
+    <VersionPrefix>0.0.1</VersionPrefix>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;net472</TargetFrameworks>
```
Comment (alistairjevans) [inline src/Autofac.Integration.ServiceFabric/Autofac.Integration.ServiceFabric.csproj]: I don't think we need to target 472 as well if we target netstandard2.0; 472 and ns2.0 should be fully compatible (unlike net461).

**Review by alistairjevans — CHANGES_REQUESTED** [review]
Just a target framework change needed, otherwise fine.

File: src/Autofac.Integration.ServiceFabric/Autofac.Integration.ServiceFabric.csproj (hunk 1 above)
Comment (alexmg) [reply src/Autofac.Integration.ServiceFabric/Autofac.Integration.ServiceFabric.csproj]: Good catch. Removed `net472` targets.

**Review by alistairjevans — APPROVED** [review]
🤺

**Pull request merged**